target/linux/lantiq/patches-6.12/160-owrt-lantiq-multiple-flash.patch

```diff
@@ -96,7 +96,7 @@ Signed-off-by: Maikel Bloemendal <openwrt@maikelenyvonne.nl>
 -	if (!ltq_mtd->map)
 -		return -ENOMEM;
 -
--	if (of_find_property(pdev->dev.of_node, "lantiq,noxip", NULL))
+-	if (of_property_present(pdev->dev.of_node, "lantiq,noxip"))
```

Comment:
does this patch even apply properly? 😅I bet the current kernel code uses of_find_property still, right?

Reply:
why wouldn't it?

Reply:
well the CI says otherwise, so apparently they do apply which leaves me wondering, why?

I assumed you just strg + f + replaced every appearance with `use_of_property_present`, which does not make sense for code that is present in upstream(kernel) but is removed downstream (in openwrt). 
unless the change was backported but the existing patch that removes the code was never updated to represent the fact.